prisma/schema.prisma

```diff
@@ -296,6 +297,10 @@ enum EnrollmentStatus {
   ACTIVE
   INACTIVE
 }
+enum ContentType{
+TYPE1
```

Comment:
Please use actual document types for the enum properties:
- PDF
- DOC
- DOCX
- VIDEO